apps/website/src/pages/docs/carousel.mdx

```diff
@@ -14,65 +14,70 @@ import * as AllExamples from 'examples';
-located below the slides, consisting of "dots" and "previous"/"next" buttons, used for changing slides.
-
-The currently shown slide can also be changed using the left/right arrow keys or by dragging on a touch device.
+The Carousel component consists of a set of slides, normally displayed one at a time. A navigation section is located below the slides, consisting of ["dots"](#dots-list) and ["previous"/"next"](#navigation) buttons, used for changing slides.
 
 ## Usage
 
-Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
+### Controlled State
+
+By default, the carousel maintains its own state. But it can also be controlled externally using `activeSlideIndex` and `onSlideChange`.
 
 <LiveExample src='Carousel.controlled.tsx'>
   <AllExamples.CarouselControlledExample client:load />
 </LiveExample>
 
-The subcomponents can be used outside of Carousel. The following example shows how `Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
+### Only Dots
+
+`Carousel.DotsList` can be used without the `Carousel` component. Here is an example to demonstrate this where clicking on the right/left half of the slide will go one slide forward/backward.
 
 <LiveExample src='Carousel.onlyDots.tsx'>
   <AllExamples.CarouselOnlyDotsExample client:load />
 </LiveExample>
 
-## Props
-
-Properties table is to be defined.
-
-<PropsTable path={frontmatter.propsPath} />
-
 ## Subcomponents
 
-This component uses a composition approach so it should be used with the provided subcomponents.
+This component uses a composition approach, similar to most of the other components. The usage of all subcomponents is not mandatory. We have taken extra care to reduce verbosity while designing the Carousel API.
 
-### Slider
+### Carousel.Slider
 
 `Carousel.Slider` is the scrollable list that should consist of `Carousel.Slide` components.
 
-### Slide
+### Carousel.Slide
 
 `Carousel.Slide` is used for the actual slide content. The content can be specified through `children`. It is recommended that the slide content bring its own dimensions (esp. height) and that the dimensions should be the same for all slides.
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Carousel/CarouselSlide.d.ts'} />
 
-### Navigation
+### Carousel.Navigation
 
-The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right, and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
+The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
```

Comment:
currently there is no easy way to get props tables for multiple components defined in the same file, but in this case i would say no props table needed, since it is a very simple component.

my only other comment is to update this text to link to each heading.

```suggestion
The `Carousel.Navigation` component by default consists of the [`PreviousButton`](#carouselnavigationpreviousbutton) and [`NextButton`](#carouselnavigationnextbutton) shown on the left and right and the [`Carousel.DotsList`](#carouseldotslist) component shown in the middle. `children` can be specified to override what is shown in this navigation section.
```